### PR TITLE
fix #13

### LINKED ISSUE #13: ERROR: Value of non-numeric column 'CLASS_LABEL' (index 26832) not specified in sparse row 160
```julia
julia> using HTTP, JSON, ARFFFiles

julia> function load_openml(id)
           r = JSON.parse(String(HTTP.request("GET", "https://www.openml.org/api/v1/json/data/$id").body))
           arff_file = HTTP.request("GET", r["data_set_description"]["url"])
           ARFFFiles.load(IOBuffer(arff_file.body))
       end
load_openml (generic function with 1 method)

julia> load_openml(390)ERROR: Value of non-numeric column 'CLASS_LABEL' (index 26832) not specified in sparse row 160
Stacktrace:
  [1] error(s::String)
    @ Base ./error.jl:33
  [2] _readcolumns_pushzero
    @ ~/.julia/packages/ARFFFiles/2Qv2c/src/ARFFFiles.jl:955 [inlined]
  [3] readcolumns(r::ARFFReader{IOBuffer}; opts_sq::Parsers.Options, opts_dq::Parsers.Options, date_opts_sq::Vector{Parsers.Options}, date_opts_dq::Vector{Parsers.Options}, maxbytes::Nothing, chunkbytes::Int64)
    @ ARFFFiles ~/.julia/packages/ARFFFiles/2Qv2c/src/ARFFFiles.jl:823
  [4] readcolumns(r::ARFFReader{IOBuffer})
    @ ARFFFiles ~/.julia/packages/ARFFFiles/2Qv2c/src/ARFFFiles.jl:666
  [5] load(::typeof(ARFFFiles.readcolumns), ::IOBuffer; opts::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
    @ ARFFFiles ~/.julia/packages/ARFFFiles/2Qv2c/src/ARFFFiles.jl:564
  [6] load
    @ ~/.julia/packages/ARFFFiles/2Qv2c/src/ARFFFiles.jl:562 [inlined]
  [7] #load#8
    @ ~/.julia/packages/ARFFFiles/2Qv2c/src/ARFFFiles.jl:569 [inlined]
  [8] load
    @ ~/.julia/packages/ARFFFiles/2Qv2c/src/ARFFFiles.jl:569 [inlined]
  [9] load_openml(id::Int64)
    @ Main ./REPL[16]:4
 [10] top-level scope
    @ REPL[17]:1
```

### PR DESCRIPTION
**Pull request by jbrea**

### CODE REVIEW

**Comment by jbrea** [comment]
@cjdoris Did you have a chance at looking at this?

File: src/ARFFFiles.jl [hunk 1]
```diff
@@ -972,11 +972,19 @@ end
     return (pos, done)
 end
 
+_zero(::Any, ::Any, ::Any, ::Any) = 0
+function _zero(::AbstractVector{<:Union{<:AbstractString, Missing}}, r, i, nrows)
+    @warn("Value of string column '$(r.colnames[i])' (index $(i-1)) is 0 in sparse row $nrows (see warning in section sparse ARFFF files https://waikato.github.io/weka-wiki/formats_and_processing/arff_developer/")
+    ""
+end
+function _zero(::CategoricalVector, r, i, nrows)
+    r.pools[r.coltypeidxs[i]][1]
```
Comment (cjdoris) [inline src/ARFFFiles.jl:981]: I believe this should be `r.pools[r.colkindidxs[i]][1]`

**Comment by jbrea** [comment]
Ah, yes, thanks.
(for unknown reasons I didn't see your comment on github).

**Comment by cjdoris** [comment]
Just tried it on some datasets, works great thanks!

**Comment by cjdoris** [comment]
I'm registering a new version now.

**Pull request merged**